installer.sh

```diff
@@ -20,19 +32,31 @@ mkdir -p $LOCATION
 
 echo "Installing cirrus to LOCATION=${LOCATION}" > ${LOCATION}/install.log
 cd $LOCATION
-# replace this with git clone of cirrus repo
-git clone ${CIRRUS_REPO} cirrus 1>> ${LOCATION}/install.log
 
-cd cirrus
+CUSTOM_PYPI_SERVER=${CIRRUS_PYPI_URL:-""}
+CIRRUS_VERSION=${CIRRUS_VERSION_OVERRIDE:-""}
+
+CIRRUS_PIP_REQ="cirrus-cli"
+if [ "x$CIRRUS_PIP_REQ" != "x" ];then
+	CIRRUS_PIP_REQ+="$CIRRUS_VERSION"
+fi
+
 # bootstrap virtualenv
 virtualenv venv
 . venv/bin/activate
-pip install -r requirements.txt 1>> ${LOCATION}/install.log
 
-# run installer
-export CIRRUS_HOME=$LOCATION/cirrus
-export VIRTUALENV_HOME=$LOCATION/cirrus/venv
-python bootstrap.py
-python setup.py develop  1>> ${LOCATION}/install.log
+if [ "x$CUSTOM_PYPI_SERVER" == "x" ];then
+    pip install --index-url=${CUSTOM_PYPI_SERVER} ${CIRRUS_PIP_REQ} 1>> ${LOCATION}/install.log
+else
+    pip install ${CIRRUS_PIP_REQ} 1>> ${LOCATION}/install.log
```

Comment:
If and else clauses seem to be switched here - I get an error without a CUSTOM_PYPI_SERVER set due to a bad url